### ISSUE TITLE
How does the prior loss behave with and without batchnorm ?

### ISSUE THREAD

**Issue by dawood95** [stevenygd/PointFlow#6]
I'm trying to train a similar model. My prior loss doesn't decrease and stays around 60k (for a vector of dimension 256).

**Comment by stevenygd** [comment]
Are you referring to the batch norm in point CNF or latent CNF?

**Comment by dawood95** [comment]
Both. If you could share loss curves, that would be very helpful.

**Comment by stevenygd** [comment]
The loss doesn't decrease without the batch norm from my end.
